src/averageAges.js

```diff
@@ -55,7 +67,14 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const filterDifferents = people.filter(({ mother, sex }) => {
```

Comment:
filterDifferents? By name, I thought that value is an array with some differents ) But it's array with children